aws_lambda_builders/workflows/nodejs_npm_esbuild/workflow.py

```diff
@@ -5,20 +5,17 @@
 import logging
```

Comment:
This file changed quite a bit. Might be easier to open the new and the old file in different windows (without the file diffs), and comparing.